build.go

```diff
@@ -33,6 +34,8 @@ import (
 	"github.com/buildpacks/pack/internal/style"
 )
 
+const lifecycleImageRepo = "natalieparellano/lifecycle-image" // TODO: change when https://github.com/buildpacks/lifecycle/pull/294 is merged.
```

Comment:
I suggest to do something (write to yourself/add it to one of the PRs/AC) so we won't forget it here.

Reply:
Ah, I was leaving this PR in "draft" status until we made the change. But you are right, it's better to call it out explicitly.